securedrop_export/disk/service.py

```diff
@@ -0,0 +1,133 @@
+import logging
+
+from securedrop_export.exceptions import ExportException
+
+from .cli import CLI
+from .status import Status
+from .new_status import Status as NewStatus
+
+logger = logging.getLogger(__name__)
+
+
+class Service:
+    def __init__(self, submission, cli=None):
+        self.submission = submission
+        self.cli = cli or CLI()
+
+    def check_connected_devices(self) -> Status:
+        """
+        Check if single USB is inserted.
+        """
+        logger.info("Export archive is usb-test")
+
+        try:
+            all_devices = self.cli.get_connected_devices()
+            num_devices = len(all_devices)
+
+        except ExportException as ex:
+            # Use legacy status instead of new status values
+            raise ExportException(sdstatus=Status.LEGACY_ERROR_USB_CHECK) from ex
+
+        if num_devices == 0:
+            raise ExportException(sdstatus=Status.LEGACY_USB_NOT_CONNECTED)
+        elif num_devices == 1:
+            return Status.LEGACY_USB_CONNECTED
+        elif num_devices > 1:
+            raise ExportException(sdstatus=Status.LEGACY_USB_ENCRYPTION_NOT_SUPPORTED)
+
```

Comment:
The current client does not expect `MULTI_DEVICE_DETECTED`, so we still report the legacy `ENCRYPTION_NOT_SUPPORTED` status here. In the new_service.py we return the new, more specific status.